src/components/SettingsDialog/ExportBoard/PrintView/PrintView.scss

```diff
@@ -82,20 +72,39 @@ $color-button-shadow: rgba(133, 144, 147, 0.5);
 }
 
 .print-view {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
   background-color: $color-white;
   overflow: visible;
-  width: 850px;
-  padding: 40px 70px;
+  padding-top: 1cm;
+  padding-right: 1cm;
+  padding-bottom: 1cm;
+  padding-left: 1.5cm;
```

Comment:
Why `1.5cm`?

Reply:
Because I thought that it's kinda standard that the left border is a little bit larger than the right one